dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/DimensionalObject.java

```diff
@@ -133,6 +135,16 @@
      */
     String getDimensionDisplayName();
 
+    /**
+     * Returns the value type of the dimension.
```

Comment:
@maikelarabori could you add a note in the Javadoc that not all implementing classes have a ValueType field, and clarify the behavior in those cases (e.g. null can be returned).

Reply:
Clarification add @larshelge, thanks

Reply:
Clarification added @larshelge, thanks